src/js/components/RangeInput/StyledRangeInput.js

```diff
@@ -108,6 +108,7 @@ const firefoxMicrosoftThumbStyle = css`
   ${rangeThumbStyle} margin-top: 0px;
   height: ${props => props.theme.global.spacing};
   width: ${props => props.theme.global.spacing};
+  ${props => props.focus && focusStyle()}
```

Comment:
I am not seeing the focus indicator on the thumb. Instead, I just see the browser default around the entire range input. it seems like the focus is still on the overall element. I am on MacOS Chrome:
<img width="1398" alt="Screen Shot 2021-05-14 at 9 41 54 AM" src="https://user-images.githubusercontent.com/12522275/118302091-a8813200-b498-11eb-84b3-1faea65397a1.png">

Reply:
Actually, I am on Windows Firefox and it's working on it but not on the Chrome. I take a look into it.

Reply:
Also,  shall I remove the browser default focus indicator or not ?